coreneuron/sim/scopmath/sparse_thread.cpp

```diff
@@ -730,9 +730,7 @@ static void re_link(SparseObj* so, unsigned i) {
 }
 
 static SparseObj* create_sparseobj() {
-    SparseObj* so = (SparseObj*) myemalloc(sizeof(SparseObj));
-    nrn_malloc_lock();
-    nrn_malloc_unlock();
+    auto* so = new SparseObj{};
```

Comment:
unified memory..

Reply:
They're no-ops anyway: https://github.com/BlueBrain/CoreNeuron/blob/4ffe156ee94a0b9a115175e446c2f7958805ba91/coreneuron/sim/scopmath/sparse_thread.cpp#L68-L69

I have some cleanup of scopmath in a separate branch, I wouldn't worry about this part too much for now.